cmd/init.go

```diff
@@ -435,6 +485,28 @@ func (o *initOptions) run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// Set hosts to /etc/hosts file
+	if o.setHosts {
+		err := setHosts(gitopsName, o)
+		if err != nil {
+			fmt.Printf("\033[33m%s\033[0m\n", err)
+		}
+	}
+
+	// Set hosts to /etc/hosts file and generate local certificates
+	if o.local {
+		err := setHosts(gitopsName, o)
+		if err != nil {
+			fmt.Printf("\033[33m%s\033[0m\n", err)
+		}
+
+		certDir, err := generateWildcardCerts(o.domain)
```

Comment:
You're duplicating this call in such a way that if a person uses `--mkcerts` or `--set-hosts` along with `--local` we get wrong behavior. Why not just make it so that `--local` will set `--mkcerts` and `--set-hosts` to true rather than fully duplicating the logic of these flags?